src/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,58 @@
+import { SortBy } from '../types/SortBy';
+import React from 'react';
+import cn from 'classnames';
+
+interface Props {
+  sortByElement: SortBy;
+  reverseVisibleGoods: boolean;
+  changeFilter: (sortBy: SortBy) => void;
+  changeDirection: (isReverse: boolean) => void;
```

Comment:
```suggestion
  onFilterChange: (sortBy: SortBy) => void;
  onSortFilterChange: (isReversed: boolean) => void;
```
In react there is a convention that `handlers` passed as prop should have prefix `on`